### ISSUE TITLE
Back button on preview

### ISSUE THREAD

**Issue by plafue** [plafue/writeily-pro#142]
... should go back to either edit mode or to the notes list, depending where the user came from

**Comment by Ridderby** [comment]
I'd say that this is valid in the edit view as well. I'd like to go back to the note-list in the folder I came from, not to the root note folder.

**Comment by GadgyRoid** [comment]
When we are editing a file and preview it, and click on the back button, it takes me not to the `NoteActivity.java` screen, but to the Fragment that displays the list of folders and files.

**Comment by kud** [comment]
Yeah, default android back button should go back and not quit the app.
